bootstrap/090_post_setup.sql

```diff
@@ -69,80 +69,109 @@ CREATE OR REPLACE TASK TASKS.PROBE_MONITORING
 DECLARE
   sql STRING;
 BEGIN
-    CALL INTERNAL.GET_PROBE_SELECT() into :sql;
```

Comment:
This was largely changed pushed down into another scripting block to make sure we always do probes and quota consumption. I'll comment on what actually changed.